app/api/fetch-email-from-source.jsx

```
@@ -1,122 +1,175 @@
+
+    return response.data.accessToken;
+  } catch (error) {
+    console.error("❌ Error refreshing token:", error);
+    throw error;
+  }
+}
+
+export const action = async ({ params}) => {
+  const days=2;
+  const sourceId = parseInt(params.sourceId, 10);
+  if (isNaN(sourceId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)